src/instructlab/configuration.py

```diff
@@ -238,7 +238,13 @@ class _generate(BaseModel):
     # additional fields with defaults
     num_cpus: PositiveInt = DEFAULTS.NUM_CPUS
     chunk_word_count: PositiveInt = DEFAULTS.CHUNK_WORD_COUNT
-    num_instructions: PositiveInt = DEFAULTS.NUM_INSTRUCTIONS
+    # DEPRECATED: see sdg_scale_factor instead
+    # Left in place so that we can still detect and give a warning if its
+    # specified in an old configuraiton file.
+    num_instructions: Optional[int] = Field(
+        -1, deprecated="see 'sdg_scale_factor' instead", exclude=True
```

Comment:
MyPy does not understand default as argument. You have to use a keyword argument here:
```suggestion
        default=-1, deprecated="see 'sdg_scale_factor' instead", exclude=True
```

Reply:
ah ha! that's what i missed with mypy. thank you!